dev/tidb-in-kubernetes/deploy/alibaba-cloud.md

```diff
@@ -206,10 +206,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
 
 通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是，`operator_helm_values` 配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
 
+- 在 `terraform.tfvars` 中设置 `operator_helm_values`：
+
 ```hcl
-variable "operator_helm_values" {
-  default = file("my-operator-values.yaml")
-}
+operator_helm_values = "./my-operator-values.yaml"
+```
```

Comment:
Please indent this code block with 4 spaces.

Reply:
done